partcad-cli/src/partcad_cli/cli_status.py

```diff
@@ -46,9 +46,7 @@ def cli_status(args):
         def get_total():
             with pc_logging.Action("Status", "total"):
                 total = (get_size(path)) / 1048576.0
-                pc_logging.info(
-                    "Total internal data storage size: %.2fMB" % total
-                )
+                pc_logging.info("Total internal data storage size: %.2fMB" % total)
```

Comment:
Somewhat related: some time ago I started to follow convention that instead of comments in code I will use debug logging which is accessible both in sources and during runtime.